src/event-handler/index.ts

```diff
@@ -37,7 +37,7 @@ export function createEventHandler<TTransformed>(
     log: createLogger(options.log),
   };
 
-  if (options && options.transform) {
+  if (options?.transform) {
```

Comment:
I had problems with optional chaining on the past. The compiled JS would continue to have it which broke lots of runtime environments. Can you check the compiled code in the pkg folder if the .? gets replaced?

Reply:
TypeScript should be transpiling it out of the box, but I'll confirm to be safe :)

Reply:
it was in the `dist-web` folder here: https://github.com/gr2m/octokit-plugin-create-pull-request/issues/68
We don't do a web build for `@octokit/webhooks` yet so we should be fine